### PR TITLE
Fix remaining mutants, part 1 (48 -> 22)

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -75,5 +75,12 @@
     "scripts": {
         "test": "phpunit --testdox --no-interaction",
         "test-watch": "phpunit-watcher watch"
-    }
+    },
+    "repositories": [
```
Comment (arogachev) [inline composer.json:79]: Need to release translator and change it back.

Comment (arogachev) [reply composer.json:79]: https://github.com/yiisoft/translator/pull/93

File: tests/Rule/IpTest.php [hunk 2]
```diff
@@ -527,16 +557,99 @@ public function dataValidationFailed(): array
...
+                    'data' => new Ip(
+                        notInRangeMessage: 'Attribute - {attribute}, value - {value}.',
+                        ranges: ['10.0.0.1', '!10.0.0.0/8', '!babe::/8', 'any'],
+                    ),
+                ],
+                ['data' => ['Attribute - data, value - 10.0.0.2.']],
+            ],
 
-            ['192.168.005.001', [new Ip()], ['' => [$message]]], // Leading zeroes are not supported
+            'leading zeroes' => ['192.168.005.001', [new Ip()], ['' => [$message]]],
```
Comment (vjik) [inline tests/Rule/IpTest.php:591]: Why don't support leading zeroes?
Suggest create issue and add support.

File: src/Rule/IpHandler.php [hunk 3]
```diff
@@ -80,116 +86,72 @@ private function getIpParsePattern(): string
...
-            );
-            return $result;
+            return self::getGenericErrorResult($rule->getNotInRangeMessage(), $context, $value);
         }
-        return $result;
+
+        return null;
+    }
+
+    private static function getGenericErrorResult(string $message, ValidationContext $context, string $value): Result
```
Comment (vjik) [inline src/Rule/IpHandler.php:150]: Why all private methods is static?

Comment (arogachev) [reply src/Rule/IpHandler.php:150]: Because these methods do not require current instance (`$this`).

File: tests/Rule/IpTest.php (hunk 2 above)
Comment (arogachev) [reply tests/Rule/IpTest.php:591]: > Why don't support leading zeroes?

I don't know, but it's out of scope of this PR.

> Suggest create issue and add support.

Created the issue - #387.

**Review by vjik — APPROVED** [review]

**Pull request merged**